client/src/pages/Paste.js

```diff
@@ -38,6 +65,14 @@ function Paste({ match }) {
           </CreatedAt>
           <Author name={paste.author} />
           <PasteArea>{paste.value}</PasteArea>
+          <EmailInput
+            placeholder="Enter the email of your recipient"
+            value={inputValue}
+            type="email"
+            onChange={event => setInputValue(event.target.value)}
+          ></EmailInput>
+          <SendEmailButton onClick={handleClickEvent}></SendEmailButton>
```

Comment:
Multiple clicks on the button resulted in multiple emails. Maybe you can add a functionality to prevent mutliple clicks like a status "disabled" or a success state for the button so it is now clickable anymore. If input value changes, it might be useful to remove the disabled state to allow another share via email.